### ISSUE TITLE
About the performance of tieredImageNet

### ISSUE THREAD

**Issue by LoveMiki** [loadder/Dynamic-Meta-filter#5]
Dear author,
I downloaded your 1-shot tieredImageNet pretrained model and implemented with the default settings (i.e., group=64). However, I can only get 66.80% by averaging 10,000 episodes. Any suggestions to get the similar performance in your paper? By the way, I use the dataset format where all photos are 84*84. Is the problem happening with the pixel size?

Thank you.

**Comment by LoveMiki** [comment]
I figure out that you are using the data format from DeepEMD where all images are 224*224.

**Comment by chmxu** [comment]
> I figure out that you are using the data format from DeepEMD where all images are 224*224.

Yes, we use the data provided by DeepEMD with 224x224 size, but we resize the images to 84x84 for all experiments. I am not sure if your setting is the same as mine.
